apps/authentication-portal/src/main/webapp/basicauth.jsp

```diff
@@ -177,6 +177,10 @@
 
         ResendCodeRequestDTO selfRegistrationRequest = new ResendCodeRequestDTO();
         UserDTO userDTO = AuthenticationEndpointUtil.getUser(resendUsername);
+        int atIndex = resendUsername.lastIndexOf('@');
```

Comment:
If we use email as the username, isn't this break the expected behavior?

Reply:
Will get resolved with https://github.com/wso2/identity-apps/pull/3921